src/tools/Tools.mjs

```diff
@@ -15,8 +15,12 @@ export default class Tools {
         }
 
         let factory = (cb, stage) => {
-            cb(null, this.createRoundRect(stage, w, h, radius, strokeWidth, strokeColor, fill, fillColor));
-        }
+            if (Utils.isWeb) {
+                cb(null, this.createRoundRect(stage, w, h, radius, strokeWidth, strokeColor, fill, fillColor));
+            } else {
+                stage.platform.createRoundRect(cb, stage, w, h, radius, strokeWidth, strokeColor, fill, fillColor);
```

Comment:
I don't see implementation of createRoundRect in platform namespace anymore.

Reply:
stage.platform.createRoundRect is now in the Lightning-Spark repo platform area.